<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="ElevatorControlSystem">
   <widget class="QGroupBox" name="PassengerSelectGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Passenger Select</string>
    </property>
    <widget class="QComboBox" name="SelectPassenger">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>&lt;Passenger&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Passenger 0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Passenger 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Passenger 2</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="PassengerWeight">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Passenger Weight</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="PassengerFloor">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Floor</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="FloorSelect">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Elevator Controls</string>
    </property>
    <widget class="QPushButton" name="Button0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="HelpButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenDoorButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;&gt;</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseDoorButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&gt;&lt;</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PersonControl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Person Controls</string>
    </property>
    <widget class="QPushButton" name="ElevatorUp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Call Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ElevatorDown">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Call Down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BlockDoorButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Block Door</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExitElevator">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EnterElevator">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="BuildingControl">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Building</string>
    </property>
    <widget class="QCheckBox" name="TogglePowerOutage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PowerOutage</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ToggleFire">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Fire</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ElevatorSelect">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Elevator Select</string>
    </property>
    <widget class="QComboBox" name="SelectElevator">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>&lt;Elevator&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Elevator 0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Elevator 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Elevator 2</string>
      </property>
     </item>
    </widget>
    <widget class="QLCDNumber" name="ElevatorFloor">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="DoorLight">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #E10600;</string>
     </property>
     <property name="text">
      <string>Door</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Floor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Door Status</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="ElevatorWeight">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Weight</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Settings">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QSpinBox" name="NumPassengers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="NumElevators">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevators MaxWeight</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="NumFloors">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="ElevatorsMaxWeight">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string># of passengers</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string># of Elevators</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string># of Floors</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ResetSimulation">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>83</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Console</string>
    </property>
    <widget class="QTextBrowser" name="Console">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
